index.bs

```diff
@@ -361,13 +361,13 @@ one line except when a line break is definitely necessary.</p>
  <pre>
  WEBVTT
 
- 00:00:00.000 --> 00:00:04.000 position:10%,start align:left size:35%
+ 00:00:00.000 --> 00:00:04.000 position:10%,left align:left size:35%
```

Comment:
OK, maybe we should use 'line-left' and 'line-right' then? Those make sense to me.